steambox_nvidia/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM steambox
+MAINTAINER Jereme Corrado "jereme@zoion.net"
+
+USER root
+
+RUN apt-get install -y debian-archive-keyring
+RUN echo "deb http://ftp.debian.org/debian/ jessie-backports main contrib non-free" > /etc/apt/sources.list.d/jessie-backports.list
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y -t jessie-backports nvidia-driver
+
```

Comment:
@rsharo Yes, you understand my change correctly.

As to the question of optimized drivers... I know very little here.  My first time using Steam, was last week - the day you received my first patch :)  I will look into it and see what I can learn.

Perhaps, for now, you could merge this PR (after I update `util/steambox` for explicity character device names) as I believe it delivers some value.  I will follow up with details on the alternate driver furnishes by Valve.

Anecdotally, I can say performance seems quit good to me.  I'm using the  DKMS built driver from the `jessie-backports` pkg, on my hosts system.  By piling on test 3d models I'm able to add enough load to see the GPU scale to its max frequency without seeing the frame rate drop below my monitors refresh rate, though it is a modest 60 Hz at 1920x1080.

This is my first machine in ages without an integrated display adapter.  The last one used the AGP bus :)